lib/datasets/tlc-trip-yellow-taxi.rb

```diff
@@ -0,0 +1,110 @@
+require "parquet"
+require_relative "dataset"
+
+module Datasets
+  class TLCTripYellowTaxi < Dataset
```

Comment:
How about `Dataset::TLC::YellowTaxiTrip`?
We'll add `Dataset::TLC::GreenTaxiTrip` and so on later.

https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page says "Yellow and green taxi trip records". So `YellowTaxiTrip` is better than `TripYellowTaxi`.

Reply:
fix: [7806239](https://github.com/red-data-tools/red-datasets/pull/151/commits/780623979373cbe1d8b321a7ad4eb9dc308a3df1)
Thank you so mcuh. It sounds so nice. I renamed TLCTripYellowTaxi to TLC::YellowTaxiTrip.